tests/Doctrine/Tests/ORM/Functional/Ticket/DDC2775Test.php

```diff
@@ -52,6 +51,8 @@ public function testIssueCascadeRemove()
 
         // With the bug, the second flush throws an error because the cascade remove didn't work correctly
         $this->_em->flush();
+
+        $this->addToAssertionCount(1);
```

Comment:
The test works around a cascade removal: what was removed? Can we try a fetch and verify that it fails?